src/components/Loading.js

```diff
@@ -22,10 +22,11 @@ const Loading = (props) => {
     return () => { clearTimeout(updateSlowLoad) }
   })
 
+  const { showSlowLoadMessage, useLoadingWrapper, ...reactLoadProps } = props;
   return (
-    <div className={props.useLoadingWrapper ? "loading-wrapper" : ""}>
-      <ReactLoading {...props} />
-      {slowLoad && props.showSlowLoadMessage ? <span>Still loading... Please, be patient.</span> : null}
+    <div className={useLoadingWrapper ? "loading-wrapper" : ""}>
+      <ReactLoading {...reactLoadProps} />
+      {slowLoad && showSlowLoadMessage ? <span>Still loading... Please, be patient.</span> : null}
```

Comment:
This is not meant to change any functionality, some error messages were popping up because `showSlowLoadMessage` and `useLoadingWrapper` were being passed as props to the `ReactLoading` component, so I just removed these 2 from the props passed.